introduction/benefits.md

```diff
@@ -0,0 +1,33 @@
+This file holds the text to accompany the [What Are the Benefits of InnerSource?](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321609) video.
+This accompanying text will appear as a part of the learning path (like [this example](https://www.safaribooksonline.com/learning-paths/learning-path-lean/9781491999738/9781491946527-/part01ch01.html)).
+
+# What Are the Benefits of InnerSource?
+
+There are many benefits to collaborating via inner source.
+Inner source gives a company a scalable strategy for **guest teams to get feature requests when they need them** without the long-term burden of maintainance.
+The company as-a-whole wins as the time of guest teams is put into code that others can use.
+
+While that result is a shining benefit of inner source, there are many benefits to hosts that receive regular inner source contributions.
+Recall that, as part of the inner source process, the product owner on the host team agrees from the outset that contributed features are good and desirable.
+Inner source allows the host team to receive **help in creating a better product** for its consumers!
+
+**Inner sourcing provides the host team a scalable strategy** for fulfilling the varying amounts of features requested from its many consumers.
+Given the fixed capacity of the full-time members of the host team, it's likely that, at times, the combined business roadmaps of its consumers will require very (or even unreasonably) large amounts of work to be done in the host team's products.
+Without inner source, this situation easily leads to a stressed, overworked team dealing with many feature requests escalated to its leaders.
+
+However, if the host team operates via inner source, the engineering resources required to build those features will appear in-proportion to their importance in the form of guest contributors.
+**Inner source becomes a force multiplier** that allows a host team to temporarily act larger than its actual size during times of high demand.
+When the demand has ended, the team throughput scales back to normal levels, all without any micromanagement of team headcount or work items.
```

Comment:
The video has @rrrutledge saying "Inner source allowed engineering time to organically flow where the organization needed it" which I think is a fantastic sentence describing what happens - there is not just more engineering time available in the entirety, it just goes to the right spot in the right time.
I'm not sure where if and where to put that, but I'll leave that comment here.

Reply:
Added it right where you commented.  I think it works there.  Your thoughts?

Reply:
I think it works well there, thanks for adding it!